packages/core/src/http.interface.ts

```diff
@@ -8,10 +8,12 @@ export interface HttpRequest extends http.IncomingMessage {
 }
 
 export type RouteParameters = Record<string, string | number>;
+export type QueryParameters = Record<string, string> | Object;
```

Comment:
`export type QueryParameters = Record<string, string | number | object >;` - IMO it is better

Reply:
yup, and `object` lowercased